src/apps/pulse/components/PreviewBuy.tsx

```
@@ -0,0 +1,239 @@
+            </div>
+          </div>
+        </div>
+
+        <div className="flex justify-between" style={{margin: 10, marginBottom: 0, fontSize: 13, color: "grey"}}>
+          <div>Details</div>
+        </div>
+
+        <div>
+          <div style={{width: 422, height: 137, background: "black", borderRadius: 10, margin: 10}}>
+            {
+              detailsEntry("Rate", `1 USD ≈ ${props.buyToken?.usdValue ? Number(1/Number(props.buyToken.usdValue)).toFixed(3) : 1.000}`, false, props.buyToken?.symbol ?? '')
```

Biome (1.9.4)

[error] Prefer constants from the standard library.

Unsafe fix: Use Math.LOG2E instead.


(lint/suspicious/noApproximativeNumericConstant)